src/Avalonia.Themes.Fluent/Accents/AccentColors.xaml.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using Avalonia.Markup.Xaml;
+using Avalonia.Media;
+using Avalonia.Platform;
+using Avalonia.Styling;
+
+namespace Avalonia.Themes.Fluent.Accents
+{
+    public class AccentColors : Styles
+    {
+        public AccentColors()
+        {
+            AvaloniaXamlLoader.Load(this);
+
+            var accentColor_prov = AvaloniaLocator.CurrentMutable.GetService<IPlatformAccentColorProvider>();
```

Comment:
use `camelCase` rather than `snake_case`

Reply:
@PieroCastillo

Reply:
> use `camelCase` rather than `snake_case`

sorry, what's that?

Reply:
Instead of "accentColor_prov" it's better to follow same naming rules, as in other places of the code, - "accentColorProv" or even "accentColorProvider"